pushmi_pullyu.gemspec

```diff
@@ -32,6 +32,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'rdf', '>= 1.99', '< 4.0'
   spec.add_runtime_dependency 'rdf-n3', '>= 1.99', '< 4.0'
   spec.add_runtime_dependency 'redis', '>= 3.3', '< 5.0'
+  spec.add_runtime_dependency 'rest-client', '~> 1.8'
```

Comment:
This is the dependency that allows the rdf gem to authenticate with username/password.